src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+conceive of them as a `syllogism` or a `judgement`, but then they would cease to
+be conceived of as objects. Instead, they would be conceived of as concepts. It
+is plausible that one would have a more accurate conception of them through a
+chemical framework, as found in Chemism, which actually takes into account their
+different determinateness. But for now, they are `mechanical objects` whose
+determinateness is identical.
+
+### The Ideality of Communication
+
+Returning to the text and progressing with it, Hegel writes that: “But even in
+communication between material objects, their determinateness spreads, so to
+speak, in a similarly ideal manner” (Hegel 1991, 716). When talking about the
+diffusion of scent amongst molecules one might get the false impression that
+`communication` is the kind of process that only works with ideal, non-material
+objects. Now, It looks as if in Hegel’s time scent was thought to be transmitted
+in a non-material way. Obviously, this is no longer the case, but Hegel’s point
+is clear: `communication` is an ideal process that occurs between material
+objects. In other words, it is not just material processes that occur between
+material objects. Here, Hegel is clearly going against a materialist/mechanistic
+worldview whereby the only processes are _material_ processes. Perhaps Hegel is
+thinking of forces and, perhaps more specifically, the force of gravity as an
+example of `communication`.
+
+At the end of this long paragraph where Hegel gives examples of `communication`,
+he expands on an important _logical_ dimension of this moment. He writes:
+
+> The formal totality of the object in general, which is indifferent to the
+> determinateness and hence is not a self-determination, makes it
+> undistinguished from the other object and thus renders the interaction
+> primarily an unimpeded continuation of the determinateness of the one in the
+> other (Hegel 1991, 716) `Communication` does not work simply because the
+> determinateness between `mechanical objects` is identical but also because
+> they are indifferent to determination. Their indifference means that the
+> identical determinateness, in other words, the determinateness that is
+> _communicated_, continues from one object to the other without any alteration
+> in that determinateness. It just continues exactly as it did from the
+> beginning. Thus, not only does `communication` begin from the identity of
+> `mechanical objects` but it also continues with that identity throughout the
+> interaction.
+
+Hegel, then, concludes this section on `communication` with an analysis of how
+it functions in the “spiritual sphere” (Hegel 1991, 716). By spiritual sphere,
+Hegel has in mind the following: “Laws, morals, rational conceptions in general”
+(Hegel 1991, 716). There are laws in a society, and these laws pervade (are
+communicated) throughout society without any obstruction or alteration. The same
```

Comment:
What if someone breaks the laws? Is that an example of their failure to be communicated?

Reply:
I think that on a mechanistic conception of society, someone could not break the law because they failed to have the law or the morality of society communicated to them. If someone breaks the law it cannot be due to ignorance, at least on a mechanistic conception of society. Because his account of communication does not allow for someone to not be determined by it - it pervades all objects without hindrance.